chain/validation/state.go

```diff
@@ -144,6 +144,15 @@ func (s *StateWrapper) SetSingletonActor(addr vstate.SingletonActorID, balance v
 			return nil, nil, errors.Wrapf(err, "set network storage market actor")
 		}
 		return &actorWrapper{*smact}, s.storage, s.flush(tree)
+	case actors.StoragePowerAddress:
```

Comment:
Yeah, I don't really like per actor / per method switches, and would probably use reflect here, but I'm not sure how that could be done here